kreuzberg/_ocr/_tesseract.py

```diff
@@ -269,7 +292,15 @@ async def process_image(
         try:
             await self._validate_tesseract_version()
             image_path, unlink = await create_temp_file(".png")
-            await run_sync(image.save, str(image_path), format="PNG")
+
+            try:
+                await run_sync(save_image.save, str(image_path), format="PNG")
+            except OSError as e:
+                if "cannot write mode" in str(e):
```

Comment:
should be 
```suggestion
                if "cannot write mode" not in str(e):
                     raise
```

Then else is not required.